old/bin/aa.pl

```diff
@@ -731,7 +731,7 @@ sub form_header {
         for my $cls (@{$form->{bu_class}}){
             if (scalar @{$form->{b_units}->{"$cls->{id}"}}){
                 print qq|<td><select data-dojo-type="dijit/form/Select" id="b_unit_$cls->{id}_$i" name="b_unit_$cls->{id}_$i">
-                                    <option></option>|;
+                                    <option>&nbsp;</option>|;
```

Comment:
What problem does the non-breaking space make here? I mean, I can imagine that you would want to add the special-case null value (_!lsmb!empty!_) here, but I don't understand the difference the nbsp makes.

Reply:
No, it is not for (!lsmb!empty!) issue, I just add the selectable blank option for business unit dropdown in AR/AP > Add Transaction and AR/AP > Sale/Purchase Invoice.
You can activate the Business Unit for AR/AP and then AR/AP > Add Transaction, you will see the business unit in the transaction line. When you accidentally select one of the business unit, you cannot select empty value again. But empty option is already there. So I add 'space' to be able to select that empty option. Since it is written in old/* code style 'default_blank' is not affecting for aa.pl

Reply:
Screen recording explanation for this can be found on below link.
https://matrix-client.matrix.org/_matrix/media/r0/download/matrix.org/TKsyKPdUaAySSAQfSPgkCEqY